tox.ini

```diff
@@ -24,6 +24,7 @@ sitepackages = False
 # To show pytest logs in console, use
 #   tox -- --log-cli-level=DEBUG
 commands =
+  python -m tests.test_semgrep setUp
```

Comment:
This is clever, but you don't need it.

Add a pytest fixture in test_semgrep and it will run it for you.